### DISCUSSION TITLE
Multicolored progress bar

### DISCUSSION THREAD

**Discussion by haoyun** [Textualize/rich#2835] [Q&A]
I come from `enlighten`. I would like to implement a multicolored progress bar as the "Testing" example in https://python-enlighten.readthedocs.io/en/stable/. However, I didn't figure out a way to achieve this. Is that possible with `rich`？

**Comment by willmcgugan** [comment]
Yes, it is possible. But there is no builtin renderable for that style of progress bar. See the docs for the Rich renderable protocol.
